apps/ai-content-generator/src/hooks/dialog/useEntryAndContentType.ts

```diff
@@ -39,13 +39,21 @@ const useEntryAndContentType = (entryId: string) => {
       const fieldType = contentType?.fields.find((field) => field.id === fieldKey)?.type;
       const isRichText = fieldType === 'RichText';
 
-      entry.fields[fieldKey][fieldLocale] = isRichText
+      const formattedText = isRichText
         ? richTextModel(updatedText)
         : updatedText
             .replace(/^\s+|\s+$/g, '')
             .replace(/^['"]+|['"]+$/g, '')
             .trim();
 
+      if (!entry.fields[fieldKey]) {
```

Comment:
This fixes the issue where we couldn't apply the output to a field if it didn't already have text in it.